src/wallet3/default_daemon_comms.cpp

```diff
@@ -261,11 +271,11 @@ namespace wallet
   }
 
   std::future<std::vector<Decoy>>
-  DefaultDaemonComms::fetch_decoys(const std::vector<int64_t>& indexes)
+  DefaultDaemonComms::fetch_decoys(const std::vector<int64_t>& indexes, bool with_txid)
```

Comment:
why is `with_txid` a parameter? just set it to true and ignore it if you dont need it.